pyro/distributions/distribution.py

```diff
@@ -83,6 +83,10 @@ def score_parts(self, x, *args, **kwargs):
         distributions should override this method to compute correct
         `.score_function` and `.entropy_term` parts.
 
+        Setting ``.has_rsample`` on a distribution instance will determine
+        whether inference engines use reparameterized samplers or the score
```

Comment:
plus something like the following?:  
gradient-based inference engines like :class:`SVI`